owning-a-smart-contract.md

```diff
@@ -0,0 +1,423 @@
+# TUTORIAL: Sophia basics - owning a smart contract and owner privileges
+
+## Tutorial Overview
+This tutorial will walk teach you how one can access restrict a method to the deployer of the smart contract itself. This is a useful tool for defining critical business logic.
+## Prerequisites
+- Installed Aeproject - For installation steps reffer to [this tutorial](smart-contract-deployment-in-aeproject.md)
+
+## 1. Prepare your environment
+
+Before we begin coding up we should start by initializing an aepp project.
+### 1.1 aeproject init
+
+Let's create a new empty folder and initialise an aepp project.
+
+```
+mkdir ownable-project
+cd ownable-project
+aeproject init
+```
+
+### 1.2 aeproject epoch
+Another thing we would require is a local working environment to develop against. Luckily `aeproject` gives us that out of the box. Just type in the following command
+
+```
+aeproject node
+```
+
+You can later stop the local node by typing in 
+```
+aeproject node --stop
+```
+
+## 2. Writing the contract without access restriction
+
+The next thing that we will do, is to write a simple Sophia smart contract. This smart contract will have two functions - public and access restricted. In this section, we will not apply an access restriction yet and will write unit tests proving that the soon to be access restricted method is still called successfully by anyone
+
+### 2.1 Write the smart contract
+Lets first write our `Restricted.aes` smart contract.
+
+First create the contract file under contracts folder
+```
+touch ./contracts/Restricted.aes
+```
+
+The following code goes inside this file:
+
+```javascript=
```

Comment:
I think usually 2 spaces are used for one indentation

Reply:
Also spaces before and after each `:` type definition are common in the core-devs sophia style